graph/src/schema/introspection.graphql

```diff
@@ -73,7 +79,8 @@ type __Directive {
   name: String!
   description: String
   locations: [__DirectiveLocation!]!
-  args: [__InputValue!]!
+  args(includeDeprecated: Boolean = false): [__InputValue!]!
+  isRepeatable: Boolean!
```

Comment:
Do we need to change anything in the Rust code to implement these new fields?

Reply:
I think it should work. We do seem to even have a test for it https://github.com/graphprotocol/graph-node/blob/363f4c52955a06acaf17bd02c04baafcd36c5e40/store/test-store/tests/graphql/introspection.rs#L587-L837